### ISSUE TITLE
Refrigerator functionality / inputs

### ISSUE THREAD

**Issue by TheHomieFox** [mauro-midolo/homeassistant_electrolux_status#40]
Hi there,
I have hte EHE6799SA fridge. All sensors that the integration offer are working great. However it would be great to have input functionality that is available through the app, such as:
- Setting fridge temperature
- Setting FlexFresh temperature
- Setting Freezer temperature
- Turning Ice Maker on/off
- Turning Quick Freeze on/off
- Turning Quick Chill on/off
- Turning Fridge Settings on/off
- Turning Ice & Water Dispenser Lock on/off
- Turning Vacation Mode on/off*
*Note, I can't see the Vacation Mode anymore in the app, but I'm pretty sure that functionality exists.
Any chance any of this can be implemented? Thanks!

**Comment by witold-gren** [comment]
It is not very hard.. 😀 but If you need do it you must discover (or share) how looks json file with data.. good example to start discover all params is use this script (it helped me discover all ability in my devices):
```
import os
import pprint
import pyelectroluxconnect
ses = pyelectroluxconnect.Session("USERNAME", "PASSWORD", region="EMEA", language = "LANGUAGE")
ses.login()
appllist = ses.getAppliances()
d = open(f"devices.json", "w")
d.write(str(appllist))
d.close()
for appliance in appllist:
    # print(ses.getApplianceConnectionState(appliance))
    p = open(f"{appllist[appliance]['alias']}-profil.json", "w")
    p.write(str(ses.getApplianceProfile(appliance)))
    p.close()
    s = open(f"{appllist[appliance]['alias']}-state.json", "w")
    s.write(str(ses.getApplianceState(appliance, rawOutput=False)))
    s.close()
    rs = open(f"{appllist[appliance]['alias']}-rawstate.json", "w")
    rs.write(str(ses.getApplianceState(appliance, rawOutput=True)))
    rs.close()
```

**Comment by TheHomieFox** [comment]
Hi @witold-gren,

Thanks for the guidance.

I ran the code as instructed (only exception is that I ignored the language part as I wasn't sure if "EN" was the correct string), and the output is as follows - not sure if that's the expected output?

`Python 3.11.3 (main, Apr  5 2023, 14:15:06) [GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> import pprint
>>> import pyelectroluxconnect
>>> ses = pyelectroluxconnect.Session("XXXXXXXXXXXXXXX@gmail.com", "XXXXXXXXXXXXXXX", region="APAC")
>>> ses.login()
>>> appllist = ses.getAppliances()
>>> d = open(f"devices.json", "w")
>>> d.write(str(appllist))
207
>>> d.close()
>>> for appliance in appllist:
...     # print(ses.getApplianceConnectionState(appliance))
...     p = open(f"{appllist[appliance]['alias']}-profil.json", "w")
...     p.write(str(ses.getApplianceProfile(appliance)))
...     p.close()
...     s = open(f"{appllist[appliance]['alias']}-state.json", "w")
...     s.write(str(ses.getApplianceState(appliance, rawOutput=False)))
...     s.close()
...     rs = open(f"{appllist[appliance]['alias']}-rawstate.json", "w")
...     rs.write(str(ses.getApplianceState(appliance, rawOutput=True)))
...     rs.close()
...
32399
21069
13086
`
